core/domain/platform_parameter_registry.py

```diff
@@ -26,16 +26,16 @@
 (config_models,) = models.Registry.import_models(
     [models.NAMES.config])
 
-DATA_TYPES = platform_parameter_domain.DATA_TYPES # pylint: disable=invalid-name
+DataTypes = platform_parameter_domain.DataTypes
```

Comment:
Again, lets not crete this variable.

Reply:
Why is this readded?

Reply:
**platform_parameter_registry.DataTypes** is used several times in **platform_parameter_registry_test.py** file. This is the reason for backend test failure. So, I have to add it again.

Reply:
But can you fix it there then? I don't think it should be added here, you will need to rather fix the tests that are failing directly.

Reply:
Then I have to change **platform_parameter_registry.DataTypes** to **platform_parameter_domain.DataTypes** in platform_parameter_registry_test.py file. 
Is that the correct way to fix this issue??

Reply:
Yep I guess so

Reply:
Done!